content_scripts/courseReverseOrder.js

```diff
@@ -30,10 +30,10 @@ const addButtonStyle = () => {
 }
 
 const addCourseReverse = () => {
-    let courseContent = document.getElementsByClassName("course-content");
```

Comment:
nice!!

Reply:
See comments on issue #101 
I think we want to revert it...